ui/apps/articles/src/pages/article-card-settings.tsx

```diff
@@ -43,8 +53,28 @@ const ArticleCardSettingsPage: React.FC<unknown> = () => {
   const { t } = useTranslation('app-articles');
   const { logger } = useRootComponentProps();
 
+  const getLicenseIcon = (type: string) => {
+    if (type === 'licenseAllRights') return <LicenseAllRights />;
+    if (type === 'licenseAttribution') return <LicenseAttribution />;
+    if (type === 'licenseNoDerivatives') return <LicenseNoDerivatives />;
+    if (type === 'licenseNoRights') return <LicenseNoRights />;
+    if (type === 'licenseNonCommercial') return <LicenseNonCommercial />;
+    if (type === 'licenseShareAlike') return <LicenseShareAlike />;
+    if (type === 'licenseSomeRights') return <LicenseSomeRights />;
+    if (type === 'LicenseWtfpl') return <LicenseWtfpl />;
```

Comment:
You can use a map of type to icon element or a switch statement to make it easier to read and cleaner.